common/com/twitter/intellij/pants/util/PantsUtil.java

```diff
@@ -353,6 +354,15 @@ public static GeneralCommandLine defaultCommandLine(@NotNull File pantsExecutabl
       pantsExecutable.getAbsolutePath()
     );
     commandLine.setExePath(pantsExecutablePath);
+
+    // Seed the subprocess invocation with the environment of a shell (TODO: which shell?).
+    try {
+      Map<String, String> shellEnv = new EnvironmentUtil.ShellEnvReader().readShellEnv();
```

Comment:
It is reading env vars from a new subshell which reloads whatever it is in `~/.bash_profile`, this would be fine. Can you kindly make sure this is indeed the behavior here like mentioned in the comment?